addons/stock/migrations/12.0.1.1/post-migration.py

```diff
@@ -101,6 +101,14 @@ def merge_stock_location_path_stock_rule(env):
             [row[1]],
             row[0],
         )
+    pull_push_rule_ids = list(set([r[0] for r in rules_to_merge]))
```

Comment:
```suggestion
    pull_push_rule_ids = tuple(set([r[0] for r in rules_to_merge]))
```
and remove the other conversion

Reply:
done